### ISSUE TITLE
Feature request - Graffiti emulation

### ISSUE THREAD

**Issue by rzookol** [c0pperdragon/Amiga-Digital-Video#27]
Would be nice to add http://wiki.icomp.de/wiki/Graffiti emulation.

**Comment by c0pperdragon** [comment]
As the solution here is just using the output of the Denise chip itself, here is no way to support different graphics cards.

**Comment by rzookol** [comment]
But graffiti isn't gfx card, it's is just converter of ~chunky data sent to amiga video port to color output. Similar to DCTV card earlier. So we would get C2P for free, and 256 colors in 320x256. Denise output should be ok as both cards (DCTV and Grafitti needs only 23pins Video port of A500/CDTV/A1200).

**Comment by c0pperdragon** [comment]
Ah, OK, so I misunderstood the issue.

**Comment by seb132** [comment]
I have a Grafitti, it's a nice card but limted range of applications (even Shapeshifter is not easy to setup).
I'd say priority is elsewhe, but agree with "nice to have". :)

**Comment by aiobofh** [comment]
Right! Just putting "ColorMaster 24" here too... If that provides more useful modes.
